source/amazon/services/prerequisites/considerations.rst

```diff
@@ -222,3 +222,10 @@ The following is an example of a valid configuration.
     </service>
 
   </wodle>
+  
+  
+Enabling dashboard visualization  
+--------------------------------
+  
+   Once the configuration is complete, you can enable the `Security Information` visualization feature in the **Wazuh Dashboard**. You can read more about it `here <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_.
```

Comment:
```suggestion
Once the configuration is complete, you can enable the corresponding Security Information Management module on the Wazuh Dashboard. To learn more, see the`modules <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_ section.
```

Reply:
Consider adding the corresponding screenshot to make it more appealing to users. For example:

![image](https://github.com/wazuh/wazuh-documentation/assets/61882981/a370f92d-5600-4878-adbf-76bac86b7290)